\chapter{Talks}

% JHP
\section*{T1. Constructing a mouse splicing atlas using single cell long-read sequencing}

\begin{center}
Jihwan Park \\
\vspace{0.3cm}
\textit{School of Life Sciences, Gwangju Institute of Science and Technology (GIST), Gwangju, Republic of Korea} \\
\end{center}

\noindent
Single-cell long-read sequencing allows the analysis of every piece of biological information from the original full-length cDNA library (e.g., single nucleotide variant, structural variant, alternative splicing patterns, fusion transcripts, transcription start site, and polyadenylation site), creating new possibilities for existing single-cell applications. However, amplification artifacts and high cost hinder the use of single-cell long-read sequencing in biomedical applications. We developed Ouro-seq, a cost-effective single-cell long-read sequencing method that efficiently removes amplification artifacts while simultaneously enriching and depleting targets genes with high specificity. Our method enables a comprehensive analysis of cell type-specific transcript isoforms, the detection of mutations at the single cell level, and targeted sequencing for hundreds of genes or cells of interest. Using Ouro-seq we have constructed a single cell splicing atlas of 12 mouse tissues. We identified a number of de novo transcript isoforms and cell type-specific splicing patterns. This is the first single-cell landscape of transcript isoforms across various tissues, providing a valuable resource for the researchers. In addition, our method enables the re-analysis of existing single-cell sequencing libraries that are stored in individual laboratories, increasing the utilization of existing single-cell genomics resources to investigate important biological questions that are difficult to answer using only short-read sequencing technologies.
\newpage

% BJH

% JKK

% JEP
\section*{T4. Fibrotic niche-enriched single-cell transcriptome uncovers novel age-associated changes in liver tissue}

\begin{center}
Jong-Eun Park \\
\vspace{0.3cm}
\textit{Graduate School of Medical Science and Engineering, Korea Advanced Institute of Science and Technology (KAIST), Daejeon, Republic of Korea} \\
\end{center}

\noindent
Aging is characterized by the gradual loss of the ability to maintain homeostasis, leading to disrupted physiological integrity and increased vulnerability to chronic disease and death. Cellular senescence has been suggested as a key factor in the aging process, with its impact exerted through the paracrine activities known as senescence-associated secretory phenotype (SASP). Single-cell genomics, as a powerful technique to understand tissue biology, has been applied to uncover age-related changes, in searching for the transcriptional identity of senescent cells in various organs. Surprisingly, many single-cell studies have failed to identify age-associated senescent cells with distinct gene expression patterns, which includes the well-characterized p16-expressing senescent cells. In this study, we developed a fibrotic niche enrichment sequencing (FiNi-seq) method, to enrich cell types previously uncaptured in scRNA-seq due to the dissociation bias. We applied this method to the liver of young and old mice, and discovered novel age-associated cell types. The transcriptomic analysis combined with spatial data revealed that these cells represent an age-associated fibrotic niche, which includes dysfunctional endothelial cells, SMOC1+ fibroblasts, and recruited immune cells. Our new single-cell analysis method can be applied to diverse tissue to reveal micro-fibrotic niche and will shed light to the understanding of the early dynamics of fibrosis.
\newpage

% HSK
\section*{T5. Writing and reading genomes: \\ Functional genomics study with CRISPR and single-cell sequencing}

\begin{center}
Heon Seok Kim \\
\vspace{0.3cm}
\textit{Department of Life Science, Hanyang University, Seoul, Republic of Korea} \\
\end{center}

\noindent
Large scale genomic studies are cataloguing thousands of genetic mutations. With the sheer number of discovered mutations, determining their phenotype and functional characterization remains an enormous challenge. Conventional CRISPR screening have been used to determine the phenotypic effects of genetic mutations by analyzing altered cellular fitness. However, these methods were not able to determine CRISPR perturbed cellsâ€™ state in detail. Single-cell CRISPR screening enabled transcriptome changes induced by CRISPR engineering. However, they had relied on short-read sequencing which are not able to detect full-length transcripts. Herein, we developed novel single-cell technologies to directly introduce mutations into the human genome and determine their transcriptional phenotype with integrated long- and short-read sequencing among individual cells. These enable introduction of more diverse genome engineering (i.e., gene KO, point mutations, gene fusions, etc.) and investigation of their effect in depth (i.e., genetic mutations, transcript isoform usage, etc.) in a single-cell resolution. This will generate rich and valuable dataset about complex phenotypes of various genetic mutations which were not attainable before with previous methods.
\newpage

% LEB

% FJT

% BL

% YZ

% DSL
\section*{T10. Epigenomic and chromosomal architectural reconfiguration in developing human frontal cortex and hippocampus}

\begin{center}
Dong-Sung Lee \\
\vspace{0.3cm}
\textit{Department of Life Science, University of Seoul, Seoul, Republic of Korea} \\
\end{center}

\noindent
The human frontal cortex and hippocampus play critical roles in learning and cognition. We investigated the epigenomic and 3D chromatin conformational reorganization during the development of the frontal cortex and hippocampus, using more than 53,000 joint single-nucleus profiles of chromatin conformation and DNA methylation (sn-m3C-seq). The remodeling of DNA methylation predominantly occurs during late-gestational to early-infant development and is temporally separated from chromatin conformation dynamics. Neurons have a unique Domain-Dominant chromatin conformation that is different from the Compartment-Dominant conformation of glial cells and non-brain tissues. We reconstructed the regulatory programs of cell-type differentiation and found putatively causal common variants for schizophrenia strongly overlap with chromatin loop-connected, cell-type-specific regulatory regions. Our data demonstrate that single-cell 3D-regulome is an effective approach for dissecting neuropsychiatric risk loci.
\newpage

% HYC

% CHS

% HJ

% HK

% VK